app/domain/authentication/authn_jwt/id_from_decoded_token_provider.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
```

Comment:
Please align the name of the file to the command class `identity_from_decoded_token_provider`

Reply:
Done